roles/brms/tasks/main.yml

```diff
@@ -1,17 +1,17 @@
 
 ---
 # This will scan linux systems for JBoss BRMS or Drools Installations
-    - name: Gather brms.kie-api-ver
+    - name: Gather jboss.brms.kie-api-ver
       raw: find / -name kie-api* 2> /dev/null | sed 's/.*kie-api-//g' | sed 's/.jar.*//g' | sort -u
-      register: kie-api-ver
+      register: jboss.brms.kie-api-ver
       ignore_errors: yes
```

Comment:
Could we add a when condition for the jboss facts, so we only run commands when that fact is being requested?
```
when: '"jboss.brms.kie-api-ver" in facts_to_collect'
```
Better to not eat up the network with request if the fact isn't being requested.